.rubocop_base.yml

```diff
@@ -70,6 +70,8 @@ Style/ParallelAssignment:
   Enabled: false
 Style/RedundantReturn:
   AllowMultipleReturnValues: true
+Style/RegexpLiteral:
+  enabled: false
```

Comment:
Capital `E`?  That's the standard way across the other settings here.  Not sure it's actually required to be capitalized, but I'd expect it is.